### PR TITLE
Scaffold: Header

### PR DESCRIPTION
**Pull request by varffs**
Include style scaffold as why not

### CODE REVIEW

**Comment by mixcas** [comment]
#1

File: src/styl/responsive/mobile.styl [hunk 1]
```diff
@@ -0,0 +1,37 @@
+// HEADER
+
+.header-title
```
Comment (mikeyrayvon) [inline src/styl/responsive/mobile.styl]: header title seems off semantically. as it is applied to menu items as well as title items

File: src/styl/responsive/mobile.styl [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  position: fixed
+  z-index: 100
+  text-align: center
+
+#header-title-venice, #header-title-post
+  left: 10%
+  width: 80%
```
Comment (mikeyrayvon) [inline src/styl/responsive/mobile.styl]: why not `left: 0; width: 100%`?

File: src/styl/responsive/mobile.styl [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+#header-title-venice
+  top: 0px
+  padding-top: $margin-basic
+
+#header-title-post
+  bottom: 0px
+  padding-bottom: $margin-basic
+
+#header-title-about, #header-title-contact
+  top: 50vh
```
Comment (mikeyrayvon) [inline src/styl/responsive/mobile.styl]: this doesn't vertically center these items. 

screen shot 2017-10-30 at 1 17 48 pm

you could `top: calc(50vh - ($itemHeight / 2))`, or `height: 100vh; text-align: center`, or use flex

File: src/styl/responsive/mobile.styl [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+#header-title-venice
+  top: 0
+  padding-top: $margin-basic
+
+#header-title-post
+  bottom: 0
+  padding-bottom: $margin-basic
+
+#header-title-about, #header-title-contact
+  top: 50vh
```
Comment (mikeyrayvon) [inline src/styl/responsive/mobile.styl]: this doesn't vertically center these items. 

screen shot 2017-10-30 at 1 17 48 pm

you could `top: calc(50vh - ($itemHeight / 2))`, or `height: 100vh; text-align: center`, or use flex

Comment (varffs) [reply src/styl/responsive/mobile.styl]: Text align doesnt work with the rotated elements. What happens is the rotate happens in the middle of the element so yes it was 1/2 the font height too low. Have fixed

**Pull request merged**